pkg/util/iptables/iptables.go

```diff
@@ -120,6 +120,56 @@ const MinCheckVersion = "1.4.11"
 const MinWaitVersion = "1.4.20"
 const MinWait2Version = "1.4.22"
 
+// no-op implemenatation of Interface
+type fake struct{}
+
+func NewFake() *fake {
```

Comment:
I still don't understand.  Why would you need to copy the interface definition.  Make util/iptables/testing depend directly on util/iptables